content/_releases/2020-08-27.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Product Update - v20.35
+description: Details of the product update v20.35 from 27th of August 2020.
+layout: rel
+version: "20.35"
+section: Product Updates
+category: product-updates
+releaseDate: 2020-08-27
+---
+
+
+## Features
+
+### Recipes
+
+With this release we have revitalised the Recipes functionality of our platform.
+It underwent changes under the hood ([the Recipes API](https://api-sparrow.elastic.io/docs/v2/#recipes-(experimental))) as well as in the UI. For now the feature is in rapid advancement/development phase.
+Not all API endpoints have their equivalent UI solution on this stage.
+
+**NOT ready**
+
+## Improvements and Updates
+
+### Copy flow
+
+After initial feedback from many customers we re-introduced back the original **Copy Flow**
+functionality. It will create an exact copy of your flow in the same workspace.
+You are presented with a pop-up to enter a new name for your copy.
+
+The **Copy Flow** functionality is now renamed into **Export Flow**. We aded more
+explanatory hints to the popup window. Please visit our [Copy and Export flow](/getting-started/copy-and-export-flow)
+documentation page for more information.
+
+### Custom Error Management
+
+After initial feedback from you, we changed our Custom Error Management icon to
```

Comment:
tautology
- After initial feedback from many customers we re-introduced
- After initial feedback from you, we changed

Reply:
removed one